libraries/load-themed-styles/package.json

```
@@ -19,5 +20,13 @@
   "devDependencies": {
     "@rushstack/heft": "workspace:*",
     "local-web-rig": "workspace:*"
+  },
+  "exports": {
+    ".": {
+      "require": "./lib/index.js",
+      "import": "./lib-es6/index.js",
```

```suggestion
      "import": "./lib-esm/index.js",
```

To match my upcoming isolated transpile PR.

---

I was concerned about possibly breaking existing imports, but I guess I already marked a breaking change, so may as well normalize.